src/components/Select.js

```diff
@@ -11,6 +11,7 @@ class Select extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
+      activeIndex: this.props.options.indexOf(this.props.selected),
```

Comment:
I don't think we'd want to set this by default. The hover styles should only display if you are actually hovering over an item.